Headlines/src/Controllers/TrendingCardsViewController.swift

```diff
@@ -423,4 +426,9 @@ class TrendingCardsViewController: UIViewController, UICollectionViewDelegate, U
             requestTrendingTopicsWithDate(newDate!)
         }
     }
+    
+    // MARK: - TabbedViewController
+    func tabbedViewControllerWasDoubleTapped() {
+        collectionView.setContentOffset(CGPoint.zero, animated: true)
+    }
 }
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Canillitapp/headlines-iOS/blob/f539ce350d0d725450191272ea6e28d081f1b565/Headlines/src/Controllers/TrendingCardsViewController.swift#L434"></span>File should contain 400 lines or less: currently contains 434</td>
    </tr>
  </tbody>
</table>